mappings/net/minecraft/world/dimension/DimensionType.mapping

```diff
@@ -1,58 +1,64 @@
 CLASS net/minecraft/unmapped/C_oxijnccf net/minecraft/world/dimension/DimensionType
-	FIELD f_bcecvbdh bedWorks Z
 	FIELD f_cdpesyot REGISTRY_CODEC Lcom/mojang/serialization/Codec;
-	FIELD f_ckygwuem ambientLight F
-	FIELD f_exnsbfbw MIN_HEIGHT I
-	FIELD f_eyjygtak MAX_COLUMN_HEIGHT I
-	FIELD f_hmvxmnvw MIN_SECTION I
-	FIELD f_iilfmdxv MAX_HEIGHT I
+	FIELD f_exnsbfbw MIN_Y I
+	FIELD f_eyjygtak MAX_Y I
+	FIELD f_hmvxmnvw FAR_BELOW_MIN_Y I
+	FIELD f_iilfmdxv Y_SIZE I
 	FIELD f_iulsvlis CODEC Lcom/mojang/serialization/Codec;
-	FIELD f_ivuvsukm logicalHeight I
-	FIELD f_jlzhdrnp minimumY I
-	FIELD f_ktrqqmye hasCeiling Z
 	FIELD f_lxemnxqx SIZE_BITS_Y I
-	FIELD f_nirqrruq coordinateScale D
-	FIELD f_ocwuqzjl ultraWarm Z
-	FIELD f_oslxdjvx fixedTime Ljava/util/OptionalLong;
-	FIELD f_rykbxwzz respawnAnchorWorks Z
-	FIELD f_sntuajfu hasSkyLight Z
-	FIELD f_vrlzepdd natural Z
+	FIELD f_mfsbixmi MOON_PHASES I
+	FIELD f_qnycowon MIN_HEIGHT I
 	FIELD f_wcytkhzw MOON_SIZES [F
-	FIELD f_woerqpsf effectsId Lnet/minecraft/unmapped/C_ncpywfca;
-	FIELD f_xbhtvssb height I
-	FIELD f_zxepzgfy infiniburn Lnet/minecraft/unmapped/C_ednuhnnn;
+	FIELD f_yopgdmky FAR_ABOVE_MAX_Y I
```

Comment:
`FAR_ABOVE_MAX_Y` not very clear either

what's it used for?

would `EXTREMELY_HIGH_Y` or `EXTREMELY_HIGH_Y_THRESHOLD` make sense?

Reply:
`FAR_ABOVE_MAX_Y` isn't used anywhere, but since it has a similar name to `FAR_BELOW_MIN_Y`, it should be named the same way